docs/modeling.md

```diff
@@ -538,8 +538,34 @@ s.solve(cp_model_probing_level = 2,
 
 See [the API documentation of the solvers](api/solvers.html) for information and links on the parameters supported. See our documentation page on [solver parameters](solver_parameters.html) if you want to tune your hyperparameters automatically. 
 
+## Accessing the underlying solver object
 
+After solving, we can access the underlying solver object to retrieve some information about the solve.
+For example in ortools we can find the number of search branches like this, expanding our previous example.
+```python
+import cpmpy as cp
+x = cp.intvar(0,10, shape=3) 
+s = cp.SolverLookup.get("ortools")
+# we are operating on the ortools interface here
+s += (cp.sum(x) <= 5)
+s.solve()
+print(s.ort_solver.NumBranches())
+```
 
+Other solvers, like Minizinc do not have the distinction between model and solver objects, like ortools ort_model and ort_solver.
```

Comment:
Maybe mention that each solver interfaces documents which solver native objects it initializes?
Might need some update in the docstring to make this somwhat more explicit. For OR-tools for example the docstring on ReadTheDocs is not as readible as it is in the code itself:

![image](https://github.com/CPMpy/cpmpy/assets/26082385/00d213fa-7047-4f4b-83df-5647fb94dd6f)

vs

![image](https://github.com/CPMpy/cpmpy/assets/26082385/ddece317-8a19-426c-a34f-e34ba31a8bd8)

Reply:
I'm not sure how to make readthedocs understand this is an enumeration

Reply:
I think it needs some indentation to be understood as a list